client/js/table-model.js

```diff
@@ -16,6 +17,26 @@ class TableModel {
   _setValue(location, value) {
     this.data[this._getCellId(location)] = value;
   }
+
+  findSum(column) {
+    let keys = Object.keys(this.data);
+
```

Comment:
No need for extra blank lines here. Generally, just use an extra blank line if you want to separate one block from another, or a title line from the block underneath it, for instance.